src/js/stories/DropdownStories.jsx

```diff
@@ -13,48 +15,114 @@ const style = {
     flexDirection: 'column',
     justifyContent: 'space-around',
     alignItems: 'center',
-    height: 400,
     padding: 10,
 };
 
-function createOptions(count) {
+const a2z = 'ABCDEFGHIJKLM NOPQRSTUVWXYZ';
+
+function createOptions(text = 'Option', asObject = false) {
     const options = [];
-    options.push('ABCDEFGHIJKLMNOPQRSTUVWXYZ');
-    for (let index = 0; index < count; index++) {
-        options.push(`Option ${options.length + 1}`);
+
+    for (let index = 0; index < 200; index++) {
+        const label = `${text} ${options.length + 1}`;
+        options.push(!asObject ? label : { label });
     }
     return options;
 }
 
+
+
 function Default() {
+    Utils.createCssRule(
```

Comment:
This is an interesting approach, but I wonder how often Storybook reloads the entire iFrame, or if we might end up creating a whole bunch of extra `<style/>` nodes as you poke around trying out different stories.